storage/backup/backup-cli/src/backup_types/epoch_ending/restore.rs

```diff
@@ -21,13 +21,13 @@ use aptos_types::{
     transaction::Version,
     waypoint::Waypoint,
 };
+use clap::Parser;
 use futures::StreamExt;
 use std::{collections::HashMap, sync::Arc, time::Instant};
-use structopt::StructOpt;
 
-#[derive(StructOpt)]
+#[derive(Parser)]
 pub struct EpochEndingRestoreOpt {
-    #[structopt(long = "epoch-ending-manifest")]
+    #[clap(long = "epoch-ending-manifest")]
```

Comment:
We should have `///` help comments above every argument telling:
1. What is it
2. Any default behavior (that isn't specified in the clap derive)